content/en/community/hackathons/2023-03-athens/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Athens 2023
+date: 2023-01-20T07:20:00+02:00
+draft: false
+collapsible: true
+description: Information on the Athens Unikraft Hackathon, March 30-31, 2023
+image: /assets/imgs/2023-03-hack-athens.jpg
+---
+
+## Athens Unikraft Hackathon
+
+Unikraft, [Nubificus](https://nubificus.co.uk/), [the High Speed Communication Networks Lab (HSCN)](http://hscnl.ece.ntua.gr/) and [the Computing Systems Lab (CSLab)](http://www.cslab.ece.ntua.gr/) from the National Technical University of Athens ([ICCS](https://www.iccs.gr/en/)/[NTUA](http://www.ntua.gr/index_en.html)) come together to organize the **Unikraft Athens Hackathon** to be held on Thursday and Friday, March 30-31, 2023.
```

Comment:
The [NTUA link](http://www.ntua.gr/index_en.html) gives a 404.

Reply:
I think https://www.ntua.gr is good enough, as it seems to redirect to https://www.ntua.gr/en if you access from abroad.